### PR TITLE
Edit entries php examples3

### PR DESCRIPTION
**Pull request by cslylla**
### Description

- I updated the following terms:
  - PHP String functions: explode(),
  - PHP String functions: strtolower(),
  - PHP String functions: substr(),
  
- I made the following changes:
  - Add Example section,
  - Fix typos.

- Closes: #2640 

### Type of Change

- Editing an existing entry (fixing a typo, bug, issues, etc)

### Checklist

- [X] All writings are my own.
- [X] My entry follows the Codecademy Docs style guide.
- [ ] My changes generate no new warnings.
- [X] I have performed a self-review of my own writing and code.
- [X] I have checked my entry and corrected any misspellings.
- [X] I have made corresponding changes to the documentation if needed.
- [X] I have confirmed my changes are not being pushed from my forked `main` branch.
- [X] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [X] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/php/concepts/string-functions/terms/explode/explode.md [hunk 1]
```diff
@@ -14,34 +14,55 @@ CatalogContent:
   - 'paths/computer-science'
 ---
 
-The `explode()` function takes a string and splits it on a given delimiter string. It returns an array of the substrings produced.
+The **`explode()`** function takes a `string` and splits it on a given delimiter `string`. It returns an `array` of the substrings produced.
```
Comment (yangc95) [inline content/php/concepts/string-functions/terms/explode/explode.md:17]:
```suggestion
The **`explode()`** function takes a string and splits it on a given delimiter string. It returns an array of the substrings produced.
```

Thanks for bolding! The string and array though aren't referring to a code snippet or actual String/Array objects so it doesn't need to be in backticks.

**Comment by cslylla** [comment]
@yangc95 Thanks for the explanation. I removed the backticks from the strtolower.md and substr.md files as well.

**Review by yangc95 — APPROVED** [review]
@cslylla Thanks for your thorough self-review! Looks good for a second review 🙂

File: content/php/concepts/string-functions/terms/strtolower/strtolower.md [hunk 2]
```diff
@@ -13,19 +13,35 @@ CatalogContent:
...
-The `string` is the input. It can be any string value.
+The `strtolower()` function takes one parameter, `$string`, as input and returns a new `string` with all the characters converted to lowercase. The `$string` can be any `string` value.
+
+## Example
+
+The following example uses the `strtolower()` function to convert a capitalized `string` to a lowercase `string`. Then the `echo` command prints this `string` to the console:
+
+```php
+<?php
+  echo (strtolower( "I Solemnly Swear That I Am To No Good"));
```
Comment (SSwiniarski) [inline content/php/concepts/string-functions/terms/strtolower/strtolower.md:32]:
```suggestion
  echo (strtolower( "I Solemnly Swear That I Am Up To No Good"));
```

File: content/php/concepts/string-functions/terms/strtolower/strtolower.md [hunk 3]
```diff
@@ -13,19 +13,35 @@ CatalogContent:
...
+?>
+```
+
+The example will result in the following output:
+
+```shell
+i solemnly swear that i am to no good
```
Comment (SSwiniarski) [inline content/php/concepts/string-functions/terms/strtolower/strtolower.md:39]:
```suggestion
i solemnly swear that i am up to no good
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@cslylla I did a review and only had one comment about a missing word in the Potter quote.

**Pull request merged**